src/client.ts

```diff
@@ -45,6 +48,11 @@ export interface TequilapiClient {
   connectionCancel (): Promise<void>,
   connectionIP (timeout?: number): Promise<ConnectionIPDTO>,
   connectionStatistics (): Promise<ConnectionStatisticsDTO>,
-  location (timeout?: number): Promise<ConsumerLocationDTO>,
-  sessionsList (): Promise<SessionDTO[]>
+
+  sessionsList (): Promise<SessionDTO[]>,
+
+  serviceList (): Promise<ServiceInfoDTO[]>,
+  serviceGet (serviceId: string): Promise<ServiceInfoDTO>,
```

Comment:
To make it more consistent with other endpoints:
```suggestion
  serviceGet (id: string): Promise<ServiceInfoDTO>,
```

Reply:
IMHO it's already consistent, because in `identityRegistration (id: string)` we use `id` just because it's more relevant versus `identityId`